<launch>
  

<node pkg="rosserial_python" type="serial_node.py" name="motorData" args="_port:=/dev/ttyACM0" />
<node pkg="pwn_bot_hardware" type="hwd_int_joy" name="hwd_int_joy"/>

</launch>
